go.mod

```diff
@@ -16,19 +16,21 @@ require (
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/stretchr/testify v1.3.0 // indirect
 	go.opencensus.io v0.19.1 // indirect
 	golang.org/x/net v0.0.0-20190311183353-d8887717615a // indirect
 	golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421 // indirect
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	k8s.io/api v0.0.0-20190222213804-5cb15d344471 // indirect
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
-	k8s.io/client-go v2.0.0-alpha.0.0.20190228174230-b40b2a5939e4+incompatible
+	k8s.io/apimachinery v0.0.0-20190311155258-f9b45bc4494d
+	k8s.io/client-go v10.0.0+incompatible
```

Comment:
Here I pinned `k8s.io/apimachinery` to `86fb29eff628`
aim to pinned the version to the release version of `kubernetes-1.13.4`

```
% cd ${GOPATH}/src/k8s.io/apimachinery
% git show-ref --dereference kubernetes-1.13.4
adc9689aa021e10172b0df1c854261e7fd84e123 refs/tags/kubernetes-1.13.4
86fb29eff6288413d76bd8506874fddd9fccdff0 refs/tags/kubernetes-1.13.4^{}
```

but `f9b45bc4494d` is now pointed to the `master` of `k8s.io/apimachinery`
which is now under `beta` stage of `kubernetes-1.14.0`, or `alpha` stage of `kubernetes-1.15.0`

```
% git describe HEAD
kubernetes-1.14.0-beta.0-25-gf9b45bc
```

same for the `k8s.io/client-go`
`v10.0.0` is stand for `kubernetes-1.13.0` initial release
but I'd to pinned version at `kubernetes-1.13.4`, to align with other packages under `k8s.io`

```
% cd ${GOPATH}/src/k8s.io/client-go
% git diff v10.0.0...kubernetes-1.13.4 --stat
 .travis.yml                              |   5 ++---
 CHANGELOG.md                             | 108 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Godeps/Godeps.json                       | 170 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------------------------------------
 README.md                                |  25 +++++++++++-----------
 dynamic/fake/simple.go                   |   1 +
 rest/config.go                           |  26 ++++++++++++++---------
 rest/config_test.go                      |   1 +
 tools/clientcmd/client_config.go         |   7 ++++---
 tools/clientcmd/client_config_test.go    |  15 +------------
 transport/config.go                      |   7 ++++++-
 transport/round_trippers.go              |  39 +++++++++++++++++++++++++++++++---
 {rest => transport}/token_source.go      |  14 ++++++-------
 {rest => transport}/token_source_test.go |   2 +-
 13 files changed, 281 insertions(+), 139 deletions(-)
```